src/PinMame/PinMameApi.cs

```diff
@@ -273,18 +275,26 @@ internal enum PinmameKeycode
 
 		internal struct PinmameDmdLevels
 		{
-			internal static readonly byte[] Wpc = {
-				20, 33, 67, 100
+			internal static readonly Dictionary<byte, byte> Wpc = new Dictionary<byte, byte> {
+				{ 0x00, 0 },
```

Comment:
I don't think `libpinmame` will ever return 0. Should be something to double check.

Reply:
Hmm I thought I saw it once, that's why I've added it, but I might be mistaken.